app/models/piece.rb

```diff
@@ -13,6 +14,22 @@ def move_to!(new_row, new_col)
     end
   end
 
+  def capture_en_passant(row_dest, col_dest)
+    @last_updated = Piece.where(game_id: game_id).order("updated_at desc").first
+    return if @last_updated.nil?
```

Comment:
You should probably return here if `@last_updated` is not a Pawn (ie. `return if @last_updated.nil? || @last_updated.type != "Pawn"`)